src/AllocatorBuffer.sol

```diff
@@ -47,6 +50,16 @@ contract AllocatorBuffer {
         emit Rely(msg.sender);
     }
 
+    // --- getters ---
+
+    function maxDeposit(bytes32, address) external pure returns (uint256 maxDeposit_) {
```

Comment:
I know they're too simple but can add tests for `maxDeposit` and `maxWithdraw`.

Reply:
Sure!

Reply:
Done.